packages/check-html-links/src/CheckHtmlLinksCli.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+
+import path from 'path';
+import chalk from 'chalk';
+
+import commandLineArgs from 'command-line-args';
+import { validateFiles } from './validateFolder.js';
+import { formatErrors } from './formatErrors.js';
+import { listFiles } from './listFiles.js';
+
+export class CheckHtmlLinksCli {
+  /** @type {{dir: string?, ignoreLinkPatterns: string[]?}} */
+  argvConfig;
+
+  constructor({ argv } = { argv: undefined }) {
+    const mainDefinitions = [
+      { name: 'ignore-link-pattern', type: String, multiple: true },
+      { name: 'dir', type: String, defaultOption: true },
+    ];
+    const options = commandLineArgs(mainDefinitions, {
+      stopAtFirstUnknown: true,
+      argv,
+    });
+    this.argvConfig = {
+      dir: options.dir,
+      ignoreLinkPatterns: options['ignore-link-pattern'],
```

Comment:
This is minor, but this works as intended : `node cli.js ../test-node --ignore-link-pattern "/images/**" "/price/*"`

while `node cli.js --ignore-link-pattern "/images/**" "/price/*" ./test-node` will ignore test-node.

In other words in case you want to put the folder to test as last argument you have to use the `--dir` option.